src/api/app/jobs/project_log_rotate_job.rb

```diff
@@ -1,41 +1,9 @@
 class ProjectLogRotateJob < ApplicationJob
```

Comment:
[Nitpick] Small point I know but can you please change the name to something like `CreateProjectLogEntry`? The name doesnt make sense anymore..